### DISCUSSION TITLE
Icon Request: icon-delta

### DISCUSSION THREAD

**Discussion by bashmish** [FortAwesome/Font-Awesome#21207] [Icon Requests]
Very useful to show differences between two numbers in financial apps.

Something like http://www.fileformat.info/info/unicode/char/0394/index.htm

**Comment by padde** [comment]
Can't you just use the unicode symbol?

**Comment by bashmish** [comment]
It's very different on Windows and Mac. Also Windows fonts look ugly.

**Comment by daggmano** [comment]
+1

**Comment by dmland** [comment]
+1

**Comment by tedeling** [comment]
+1

**Comment by jnnfrcyl** [comment]
+1

**Comment by ahollingsworth-r1** [comment]
+1

**Comment by DaBlick** [comment]
+1

**Comment by rwsansom** [comment]
+1

**Comment by tfitzpatrick** [comment]
+1

**Comment by stephanvierkant** [comment]
See #2828 (Icon Request: greek alphabet). I suggest to close this request, since this is kind of a duplicate request.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
